src/storage/engine/rocksdb.rs

```diff
@@ -39,8 +44,9 @@ impl EngineRocksdb {
             _ => (path.to_owned(), None),
         };
         Ok(EngineRocksdb {
-            db: try!(rocksdb::new_engine(&path, cfs)),
+            db: Arc::new(try!(rocksdb::new_engine(&path, cfs))),
```

Comment:
I prefer not to do it. There are still cases we create a DB but do not need it to be shared between threads. For example, `tikv-dump`.